_data/sites.json

```diff
@@ -12461,6 +12461,17 @@
         ]
     },
 
+    {
+        "name": "OpenTable",
+        "url": "https://help.opentable.com/s/article/How-do-I-unsubscribe-from-OpenTable-emails",
+        "difficulty": "hard",
+        "notes": "If you are using the IOS app, you can delete your account via Account Settings ⮕ Your details ⮕ Delete my account. Otherwise, you will need to contact OpenTable on <a href=\"help.opentable.com\">help.opentable.com</a> or through email <a href\"mailto:privacy@opentable.com\">privacy@opentable.com</a> and request account deletion",
```

Comment:
```suggestion
        "url": "https://help.opentable.com",
        "difficulty": "hard",
        "notes": "If you are using the iOS app, you can delete your account via Account Settings ⮕ Your details ⮕ Delete my account. Otherwise, you will need to contact OpenTable on their help page or via email",
        "email": "privacy@opentable.com",
```